### DISCUSSION TITLE
How to always show indentscope?

### DISCUSSION THREAD

**Discussion by fannheyward** [nvim-mini/mini.nvim#51] [Q&A]
Currently, `mini.indentscope` only display indent symbol on cursor line, how to show indentscope on every line?

**Comment by echasnovski** [comment]
I don't think I follow. Could you, please, elaborate?
Do you mean something along the lines "show union of all vertical lines as if cursor was placed on every visible line"? So, something like in [lukas-reineke/indent-blankline.nvim](https://github.com/lukas-reineke/indent-blankline.nvim) but for actually present indents instead of what they should be?

**Reply by fannheyward** [reply]
Currently, `mini.indentscope` only show indent line for cursor line/block. What I want is: show indent line for every block even my cursor not in it. Here is a screenshot, how to show indent line for the red area?

<img width="354" alt="x" src="https://user-images.githubusercontent.com/345274/159111524-bb46421b-c031-495f-834f-dbb6cb5f53bf.png">

By default, [lukas-reineke/indent-blankline.nvim](https://github.com/lukas-reineke/indent-blankline.nvim) will show indent line for this.

**Reply by echasnovski** [reply]
Ok, so indeed like 'indent-blankline.nvim', but for actually present indents. At the moment this can't be done and I don't have plans to implement it in the near future. 'mini.indentscope' can somewhat work alongside 'indent-blankline.nvim', though, and I want to make it work seamlessly (not possible with current Neovim functionality).

**Reply by kskarthik** [reply]
Would be nice to have a config option to show indent lines always :smile:

**Reply by echasnovski** [reply]
> Would be nice to have a config option to show indent lines always smile

This is not planned. See #99.
